Drop after FreeBSD 12.0 EOL. See https://reviews.llvm.org/rL333251

../src/modules/backlight.cpp:188:53: error: no viable constructor or deduction guide for deduction of template arguments of 'optional'
  previous_best_ = best == nullptr ? std::nullopt : std::optional{*best};
                                                    ^
/usr/include/c++/v1/optional:683:41: note: candidate template ignored: couldn't infer template argument '_Tp'
    _LIBCPP_INLINE_VISIBILITY constexpr optional(nullopt_t) noexcept {}
                                        ^
/usr/include/c++/v1/optional:689:24: note: candidate template ignored: couldn't infer template argument '_Tp'
    constexpr explicit optional(in_place_t, _Args&&... __args)
                       ^
/usr/include/c++/v1/optional:703:15: note: candidate template ignored: couldn't infer template argument '_Tp'
    constexpr optional(_Up&& __v)
              ^
/usr/include/c++/v1/optional:710:24: note: candidate template ignored: couldn't infer template argument '_Tp'
    constexpr explicit optional(_Up&& __v)
                       ^
/usr/include/c++/v1/optional:579:7: note: candidate template ignored: could not match 'optional<_Tp>' against 'waybar::modules::Backlight::BacklightDev'
class optional
      ^
/usr/include/c++/v1/optional:681:41: note: candidate template ignored: could not match 'optional<_Tp>' against 'const waybar::modules::Backlight::BacklightDev'
    _LIBCPP_INLINE_VISIBILITY constexpr optional(const optional&) = default;
                                        ^
/usr/include/c++/v1/optional:682:41: note: candidate template ignored: could not match 'optional<_Tp>' against 'const waybar::modules::Backlight::BacklightDev'
    _LIBCPP_INLINE_VISIBILITY constexpr optional(optional&&) = default;
                                        ^
/usr/include/c++/v1/optional:718:5: note: candidate template ignored: could not match 'optional<type-parameter-0-1>' against 'const waybar::modules::Backlight::BacklightDev'
    optional(const optional<_Up>& __v)
    ^
/usr/include/c++/v1/optional:726:14: note: candidate template ignored: could not match 'optional<type-parameter-0-1>' against 'const waybar::modules::Backlight::BacklightDev'
    explicit optional(const optional<_Up>& __v)
             ^
/usr/include/c++/v1/optional:736:5: note: candidate template ignored: could not match 'optional<type-parameter-0-1>' against 'const waybar::modules::Backlight::BacklightDev'
    optional(optional<_Up>&& __v)
    ^
/usr/include/c++/v1/optional:744:14: note: candidate template ignored: could not match 'optional<type-parameter-0-1>' against 'const waybar::modules::Backlight::BacklightDev'
    explicit optional(optional<_Up>&& __v)
             ^
/usr/include/c++/v1/optional:680:41: note: candidate function template not viable: requires 0 arguments, but 1 was provided
    _LIBCPP_INLINE_VISIBILITY constexpr optional() noexcept {}
                                        ^
/usr/include/c++/v1/optional:696:24: note: candidate function template not viable: requires at least 2 arguments, but 1 was provided
    constexpr explicit optional(in_place_t, initializer_list<_Up> __il, _Args&&... __args)
                       ^

--- src/modules/backlight.cpp.orig	2019-08-08 10:25:31 UTC
+++ src/modules/backlight.cpp
@@ -11,6 +11,13 @@
 
 #include <fmt/format.h>
 
+#if defined(_LIBCPP_VERSION) && _LIBCPP_VERSION < 7000
+_LIBCPP_BEGIN_NAMESPACE_STD
+template<class T>
+    optional(T) -> optional<T>;
+_LIBCPP_END_NAMESPACE_STD
+#endif
+
 namespace {
 class FileDescriptor {
  public:
